ethers-middleware/tests/gas_escalator.rs

```diff
@@ -30,9 +30,9 @@ async fn gas_escalator_live() {
     let tx = TransactionRequest::pay(Address::zero(), 1u64).gas_price(10_000_000);
 
     // broadcast 3 txs
-    provider.send_transaction(tx.clone().nonce(nonce), None).await.unwrap();
-    provider.send_transaction(tx.clone().nonce(nonce + 1), None).await.unwrap();
-    provider.send_transaction(tx.clone().nonce(nonce + 2), None).await.unwrap();
+    drop(provider.send_transaction(tx.clone().nonce(nonce), None).await.unwrap());
```

Comment:
I think `let _ = ` is more idiomatic than `drop()`

Reply:
`let _ = impl Future;`  produces [`clippy::let_underscore_future`](https://rust-lang.github.io/rust-clippy/master/#let_underscore_future)